project/Dependencies.scala

```diff
@@ -52,16 +41,11 @@ object Dependencies {
     )
   )
 
-  val Ftp = Seq(
+  val Jms = Seq(
     libraryDependencies ++= Seq(
-      "commons-net"           % "commons-net"          % "3.5",             // ApacheV2
-      "com.jcraft"            % "jsch"                 % "0.1.54",          // BSD-style
-      "org.apache.ftpserver"  % "ftpserver-core"       % "1.0.6"    % Test, // ApacheV2
-      "org.apache.sshd"       % "sshd-core"            % "1.3.0"    % Test, // ApacheV2
-      "com.google.jimfs"      % "jimfs"                % "1.1"      % Test, // ApacheV2
-      "org.slf4j"             % "slf4j-api"            % "1.7.21"   % Test, // MIT
-      "ch.qos.logback"        % "logback-classic"      % "1.1.7"    % Test, // Eclipse Public License 1.0
-      "ch.qos.logback"        % "logback-core"         % "1.1.7"    % Test  // Eclipse Public License 1.0
+      "javax.jms" % "javax.jms-api" % "2.0.1", // CDDL + GPLv2
```

Comment:
We are against anything GPL (yeah, I know this is standard JEE API) and the safe bet would be to define this as "provided" and document that user must add the dependency. Spring is also doing that: https://mvnrepository.com/artifact/org.springframework/spring-jms/4.3.4.RELEASE